code/modules/deathmatch/deathmatch_mapping.dm

```diff
@@ -0,0 +1,72 @@
+GLOBAL_LIST_EMPTY(deathmatch_points)
+
+/area/deathmatch
+	name = "Deathmatch Arena"
+	requires_power = FALSE
+	has_gravity = STANDARD_GRAVITY
+	area_flags = UNIQUE_AREA | UNIQUE_AREA
+
+/area/deathmatch/fullbright
+	static_lighting = FALSE
+	base_lighting_alpha = 255
+
+/obj/effect/landmark/deathmatch_map_spawner
+	name = "Deathmatch Map Spawner"
+	var/list/map_bounds
+
+/obj/effect/landmark/deathmatch_map_spawner/Initialize(mapload)
+	. = ..()
+	GLOB.deathmatch_points += src
+
+/obj/effect/landmark/deathmatch_map_spawner/Destroy()
+	if(map_bounds)
+		for(var/turf/to_delete in block(
+			locate(
+				map_bounds[MAP_MINX],
+				map_bounds[MAP_MINY],
+				map_bounds[MAP_MINZ],
+			),
+			locate(
+				map_bounds[MAP_MAXX],
+				map_bounds[MAP_MAXY],
+				map_bounds[MAP_MAXZ],
+			)
+		))
+			to_delete.empty()
+			to_delete.baseturfs = /turf/baseturf_bottom //if you run this outside CC z-level i will kick you in the nuts
```

Comment:
Why risk it, why make this (very dangerous) a landmark anyways? Why not use lazy loading templates?

Reply:
this is a draft